ocfweb/stats/semester_job.py

```diff
@@ -0,0 +1,109 @@
+from datetime import date
+from datetime import datetime
+from datetime import timedelta
+
+import ocflib.printing.quota as quota
+from django.http import HttpResponse
+from matplotlib.figure import Figure
+from matplotlib.ticker import MaxNLocator
+from ocflib.printing.quota import WEEKDAY_QUOTA
+from ocflib.printing.quota import WEEKEND_QUOTA
+
+from ocfweb.component.graph import canonical_graph
+from ocfweb.component.graph import plot_to_image_bytes
+
+ONE_DAY = timedelta(days=1)
+MONTH_INDEX = 1
+SEMESTER_CUTOFF_MONTH = 8
+
+year = datetime.today().year
+spring_start = date(year, 1, 1)
+fall_start = date(year, 8, 1)
+
+
+def get_start_end():
+    """reutnr a tuple of startdate and endate of the semester based on today"""
+    day = datetime.today().timetuple()
+    if day[MONTH_INDEX] < SEMESTER_CUTOFF_MONTH:
+        return (spring_start, fall_start - ONE_DAY)
+    else:
+        return (fall_start, spring_start - ONE_DAY)
```

Comment:
I find a function (current_semester_start) that gives me the start date for my current semester. I could use it in my semester job graph replacing (get_start_end). But you also mention that function like get_start_end may be useful for ocflib as well. So I am thinking to implement function (semester_dates) like you suggested in ocflib and use that function for my chart instead. How is that?  

I am assuming function (semester_dates) takes into a day(datetime object) and returns the start, end day tuple of that day's semester regardless whether that day is in the semester. Is that right?
If so, my implmentation is probably like 
```python
year = datetime.today().year
spring_start = date(year,1, 1)
fall_start = date(year, 8, 1)
if day.month < 8:
   return (spring_start, fall_start - ONE_DAY)
else:
   return (fall_start, spring_start - ONE_DAY)